eng/pipelines/extra-platforms/runtime-extra-platforms-ioslike.yml

```diff
@@ -81,15 +81,14 @@ jobs:
         - template: /eng/pipelines/common/templates/runtimes/test-variables.yml
           parameters:
             testGroup: innerloop
-      postBuildSteps:
-        - template: /eng/pipelines/common/templates/runtimes/build-runtime-tests-and-send-to-helix.yml
-          parameters:
-            creator: dotnet-bot
-            compileOnHelix: true
-            interpreter: true
-            testBuildArgs: /p:ArchiveTests=true /p:DevTeamProvisioning=- /p:RunAOTCompilation=true /p:MonoForceInterpreter=true /p:BuildTestsOnHelix=true
-            testRunNamePrefixSuffix: Mono_$(_BuildConfig)
-            extraHelixArguments: /p:NeedsToBuildAppsOnHelix=true
+      extraStepsTemplate: /eng/pipelines/common/templates/runtimes/build-runtime-tests-and-send-to-helix.yml
```

Comment:
We moved from extraStepsTemplate/extraStepsParameters in https://github.com/dotnet/runtime/pull/92375, is there a particular reason as to why we are moving back?

Reply:
Thanks, reverted. I assume this was a merge conflict issue.